src/utils/gaa.js

```diff
@@ -302,10 +302,12 @@ let GoogleUserDef;
 
 /**
  * Returns true if the URL contains fresh Google Article Access (GAA) params.
+ * @param {string=} opt_urlParams
```

Comment:
Could you rename this to `optionalQueryString`? https://google.github.io/styleguide/jsguide.html#naming-parameter-names

Reply:
Resolving since function is updated.